app/services/geo-location.ts

```diff
@@ -2,31 +2,32 @@ import Service from '@ember/service';
 import { tracked } from '@glimmer/tracking';
 
 export default class GeoLocationService extends Service {
-  @tracked status;
-  @tracked latitude;
-  @tracked longitude;
+  @tracked status: string;
+  @tracked latitude: number;
+  @tracked longitude: number;
 
   constructor() {
     super(...arguments);
 
     this.status = 'waiting';
-    this.latitude = null;
-    this.longitude = null;
+    this.latitude = 0;
```

Comment:
if you enable class props in babel you do not need to do this constructor calling

it would be something like

```
@tracked
status = 'waiting';
```

the super call in the constructor with octane will also fail linting right now as there is a bug we cannot do anything about where it expects 'owner' as the first arg 🙄